Date/Span.php

```diff
@@ -1106,7 +1106,7 @@ function setDefaultInputFormat($format)
      * @static
      * @see      Date_Span::setDefaultInputFormat(), Date_Span::getDefaultFormat()
      */
-    function getDefaultInputFormat()
+    static function getDefaultInputFormat()
```

Comment:
It's the same problem and the solution I mention in the previous comment thread.